packages/block-library/src/list-item/block.json

```diff
@@ -26,7 +26,7 @@
 			"gradients": true,
 			"link": true,
 			"__experimentalDefaultControls": {
-				"background": true,
+				"background": false,
 				"text": true
 			}
```

Comment:
This way of writing certainly works, but I feel it's hard to know whether the background is enabled or not.

In general, I think it's better to write it like this:

```
"color": {
	"gradients": true,
	"link": true,
	"background": true,
	"__experimentalDefaultControls": {
		"text": true
	}
},
```

Also, once this PR is merged, you may want to change this part of the table in #43245 to "Optional":

![list-item](https://github.com/user-attachments/assets/572e5c1e-5864-4baa-8619-f2157e4410be)

Reply:
> This way of writing certainly works, but I feel it's hard to know whether the background is enabled or not.

Agreed, it would be much better from an extender's perspective to have each support clearly written out, even if some are enabled by default.

Reply:
Will do, thanks.